gnome-initial-setup/gis-driver.h

```diff
@@ -52,11 +53,6 @@ struct _GisDriverClass
   void (* locale_changed) (GisDriver *driver);
 };
 
-typedef enum {
-  GIS_DRIVER_MODE_NEW_USER,
-  GIS_DRIVER_MODE_EXISTING_USER,
-} GisDriverMode;
-
```

Comment:
This patch can probably be upstreamed too

Reply:
It's not necessary. I split this to a different file so it could be referenced from `gis-assistant.h` without introducing a circular dependency between `gis-assistant.h` and `gis-driver.h`.

Reply:
Makes sense